feature-utils/poly-analysis/utils/status.js

```diff
@@ -0,0 +1,23 @@
+export function createSuccessStatus() {
+    return {
+        name: "SUCCESS",
+        isSuccess: true,
```

Comment:
Also, isn't this a bit redundant?

Reply:
Probably needs a closer refactor, but like I said, better not to move code around and do major refactoring at the same time.

Reply:
Right, I finally accepted this as is, and leave changes for later.